docs/extensions/info-xml.md

```diff
@@ -287,6 +288,51 @@ Elements acceptable within `<requires>`
 | -- | -- |
 | [`<ext>`](#ext) | 1+ |
 
+### `<tags>` {:#tags}
+
+* Containing element: [`<extension>`](#extension)
+* Description: Freeform tags allow extensions to be organized into categories or selected as subgroups. Tags MUST contain only letters, numbers, dashes,and colons. Tags are case-sensitive. Tags MUST follow the structure `<prefix>:<some-tag-name>`. Tags SHOULD be documented in the table below. (*If you need to use a new/unknown tag, then please submit a [documentation update](https://docs.civicrm.org/dev/en/latest/documentation/) with a description.*)
+* Contains: elements
+* Example:
+
+    ```xml
+    <extension key="org.civicrm.foo" type="module">
```

Comment:
we don't recommend this style of key now do we?

Reply:
All the other examples in the page use the Java style. I literally copied that line from the example right above. ✂️📋 

AFAIK, we've entered a grey transitional period where several people like the aesthetics of simpler names, and they seem to work in common cases, but no one can quite certify that  simpler names *always* work.

Reply:
ok - let's ignore for now